pkg/sessions/redis/testdata/cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDFzCCAf+gAwIBAgIRANg8BXFjM6eJoJ6LukqgqzQwDQYJKoZIhvcNAQELBQAw
+EjEQMA4GA1UEChMHQWNtZSBDbzAeFw0yMTA4MDkxMTE4NDVaFw0yMjA4MDkxMTE4
+NDVaMBIxEDAOBgNVBAoTB0FjbWUgQ28wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAw
+ggEKAoIBAQDDTBOJAW9wb+JNB78H9z9n6Y9uGqRpI929V8A65fKu4pnpa/0yaPhv
+wB4klr2w65uZ75ZoPoQ1BXeQQP9pLJJ/CJdHOalrvW673MsuWWjevmbCytnFTMDE
+bMtScsjXV30taE6+ps81EjrI5Sljr5YggdOuJOkXclIezmcHprbQC8nJHRyXocdk
+XQiuwKlEuLhDZkHOEz7HronIXWGXoZV/Q3JR29/I5t8oQsOk4qDL6PjuGtwV7gfA
+w34ugr+e/bD6UD6LDD9a8iLelNMXMHCpWiyFoo3mekvPqJzrlewXaAKehHLNQnif
+NfXsnxIBDjqCVfr4Ltfs++sHQfufhkynAgMBAAGjaDBmMA4GA1UdDwEB/wQEAwIC
+pDATBgNVHSUEDDAKBggrBgEFBQcDATAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQW
+BBRSnn/Lp9/aNRUpWXwLUsWKq16gvzAPBgNVHREECDAGhwR/AAABMA0GCSqGSIb3
+DQEBCwUAA4IBAQAe2wB2HbCqSGWn5ziAyzhmzQG+zP2bG93GviVaLxevk980DYVK
+d6epolyBJEUTQmvImkmJxozyPi83O3YALNw39ajioNEtCHMLx/EhyGucsJmJ9LFC
+0w3Mbs6ymmLlDFSy6dRCPiDl7SFq1H70WIqZ6R6fW0iZh+ZFc5AqHM4wu9Oa3gRX
+ztChBAKFH122cCryeDLTYVvq/dGQSFOIIb244kFHmzZDPK2eVywaIOmWDp+fkTNN
+yGlXSv/I59HTTQ6VvIZ7uvf4E49kBua4uR/TNm/ra7EJK++6lCslzLL95a1xzxLq
+2BLYzEdRwARZAtHadmVxioLR9bX8017Xs6FS
+-----END CERTIFICATE-----
```

Comment:
I'm sure somewhere in our tests we either already have a cert checked in, or we have a utility for generating certs for use in testing. Would prefer we don't duplicate, but I can't remember where we put that right now.
Will make a note to look for that and get back to you

Reply:
I think we should aim to do something like https://github.com/oauth2-proxy/oauth2-proxy/blob/42475c28f7f235c08777a24027fe8ec170763482/pkg/http/http_suite_test.go#L36-L69

Maybe we should make a test utils package that does this for us 🤔